source/nodejs/adaptivecards-designer/src/adaptive-card-schema.ts

```diff
@@ -147,6 +147,7 @@ export const adaptiveCardSchema =
           "$ref": "#/definitions/Actions"
```

Comment:
@JeanRoca 
From my understanding, we only support one Carousel. 
1. So the best user experience is that we have a button to convert an AC card to Carousel or convert it back to a normal card.
So a Carousel in Containers menu doesn't make sense to me.
![image](https://user-images.githubusercontent.com/6290692/179062230-f04c24cc-5276-4c32-8dea-903ac7f072af.png)

2. When the AC card is Carousel, we add CarouselPage to the contains, then user can drag the page to Carousel.
![image](https://user-images.githubusercontent.com/6290692/179063251-6f546b7f-3ad2-49be-ad42-7abab0a14887.png)

Reply:
The Carousel control mimics the behavior of similar elements that only allow one type of child (Table and TableRow, ColumnSet and Column, ImageSet and Image).

For #1, what would the flow be if someone had a carousel with multiple pages w/ content, and converted it back to an AdaptiveCard?

Reply:
@licanhua @anna-dingler we recently got an ask of allowing certain elements (like text blocks for example) to be part of the parent Adaptive Card and not part of the carousel page. This would enable a behavior where the title remains static and paginating through the carousel does not move the title out of place.

If we went with the proposed solution in 1 above, would that cause challenges when/if we try to enable the static element functionality later?

Reply:
@JeanRoca I think it depends how complex the static element would be. If it was just a TextBlock (or something simple), the carousel could potentially have a `title` option. However, if we want more complex static elements, it could get tricky. 

For instance, say you converted the card to a carousel, then dragged all of the static content to the card. Accidentally clicking the carousel button and then re-enabling it would move all of the static content into the carousel.

Reply:
The carousel _is_ container so i agree with the current implementation of it being part of the containers, but we should prevent the user from adding a carousel within a page, if possible. 

@anna-dingler and @JeanRoca - please discuss @JeanRoca's comment with @jwoo-msft. I believe that the experience that @JeanRoca is describing will be properties on the carousel implementation (so the carousel won't take the whole AC space) but this can also be implemented as two types of carousels.

Reply:
@paulcam206 Did you have any thoughts about this?

Reply:
I think I get @licanhua's argument, but as @anna-dingler notes, `Carousel`'s behavior most closely maps to `ColumnSet`. `Carousel` is a container solely of `CarouselPage` elements (as `ColumnSet` only has `Column` children).

The thing that makes `Carousel` seem odd is the fact that it's a singleton element. I think if anything this points out that we should have a better answer for documentation in the Designer (and make sure we have good docs for `Carousel`).

Reply:
At any rate, I think this model makes sense for now...

Reply:
I discussed this with @KesemZ, and we are going to stick with the current implementation for now since it is consistent with the current design and would be a major refactor. We can circle back to this when we make more overall changes to the designer.